<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomRoutesUi</class>
 <widget class="QDialog" name="CustomRoutesUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Routes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Read File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Routes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Unidentified nodes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomRoutesUi</receiver>
   <slot>OnReadFileButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomRoutesUi</receiver>
   <slot>OnSaveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnReadFileButton()</slot>
  <slot>OnSaveButton()</slot>
 </slots>
</ui>
